docs/version0.27.md

```diff
@@ -34,7 +34,10 @@ The following new features and notable changes since v 0.26.0 are included in th
 
 ### New `-XX:[+|-]AdaptiveGCThreading` option added
 
-To optimize performance, you can now enable adaptive threading to automatically tune the number of active parallel garbage collection (GC) threads. When enabled, the GC thread count is dynamically adjusted from collection cycle to cycle to account for changes in the workload and consider the available resources. When parallel workloads decrease, less threads can be used, reducing the overhead and freeing up resources for other processing activities.
+Adaptive threading is enabled by default, which automatically tunes the number of active parallel garbage collection (GC) threads.
+When enabled, the GC thread count is dynamically adjusted from collection cycle to cycle to account for changes in the workload and consider
+the available resources. When parallel workloads decrease, less threads can be used, reducing the overhead, freeing up resources for other
```

Comment:
The last sentence in this paragraph is a little long and would read better if broken up. Something like: 

When parallel workloads decrease, less threads are used, which reduces the overhead. Resources are freed up for other processing activities, which effectively reduces GC pause times.

Reply:
Updated, except I used "fewer threads are used".

Reply:
The last sentence isn't really explaining why GC pauses may be lower.... Something like this is more correct:


When enabled, the GC thread count is dynamically adjusted from collection cycle to cycle to account for changes in time parallel threads spend doing useful GC work (like object graph traversal) vs time synchronizing among themselves.

Reply:
I'll put my comment here rather than via the review so it's easier to see the thread of comments. I'd just make some tweaks to the latest suggestion:

"When this feature is enabled, the GC thread count is dynamically adjusted from collection cycle to cycle to account for changes in the amount of time that parallel threads spend doing useful GC work (such as object graph traversal) compared to time spent synchronizing among themselves."

Reply:
The new text suggested was already partially there in a previous sentence. I've updated it with the suggested modifications, and re-jigged the last two sentences.

Reply:
It's ok, although I was trying to avoid word 'workload'. It may mislead the reader we are talking about application parallel workload, while we are working about GC specific work (parallel roots processing, object graph traversal,...)

Reply:
I've used your words to remove "workload" and better describe the GC work.

Reply:
ok. 

you only need once to go into details what GC work is (root processing, object graph traversal....)

Reply:
Going point, fixed it up.